src/main/js/main_html5.js

```diff
@@ -510,12 +511,13 @@ require("../../../html5-common/js/utils/environment.js");
         var playPromise = executePlay(false);
         var originalUrl = _video.src;
         if (playPromise) {
-          playPromiseSupported = true;
+          ignoreFirstPlayingEvent = true;
```

Comment:
Just wondering why this is set to true here. I'm not sure I follow this part of the flow.

Reply:
There were some sync issues with when the <browser playing event> is fired with regards to the muted autoplay workflow. To get around this, I need the ```PLAYING``` notification from the video plugin to be fired from the play promise. To avoid two ```PLAYING``` notifications, I added the ```playPromiseSupported``` variable so that the <browser playing event> does not fire an additional PLAYING notification to VTC. 

This is what caused this bug PLAYER-3792. To fix this, I added the ```ignoreFirstPlayingEvent``` variable so that, if true, we do not fire the ```PLAYING```notification in the browser playing event callback. We set ```ignoreFirstPlayingEvent``` to true when starting playback with play promises. This will remain false if playback was started from a seek event, for example.

Reply:
Ok, got it. Sounds good